src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java

```diff
@@ -40,4 +44,27 @@ public boolean isSubmitLinkVisible() {
         }
         return isModuleVisible() && isLinkVisible;
     }
+
+    public boolean isEnableTestModeButtonVisible() {
+        boolean isButtonVisible = false;
+        try {
+            wait.forElementVisible(enableTestModeButton, 5, 1);
+            isButtonVisible = true;
+        } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)